packages/api/lambdas/sf-event-sqs-to-db-records/write-granules.js

```diff
@@ -453,7 +454,8 @@ const _writeGranule = async ({
   await _writeGranuleFiles({
     files,
     granuleCumulusId,
-    granuleId: granule.granuleId,
+    granule,
+    workflowError: error,
```

Comment:
Nit: _writeGranuleFiles's docstring is out of date.   Presumably from a prior PR in this sequence...?